packages/dai-plugin-mcd/src/ManagedCdp.js

```diff
@@ -122,12 +126,24 @@ export default class ManagedCdp {
     return this._cdpManager.wipe(this.id, amount, { promise });
   }
 
+  wipeAll({ promise } = {}) {
```

Comment:
these methods that are part of the class's public API shouldn't need to have `{ promise } = {}` in their arguments--it would only be necessary if you were expecting some other part of the code to call this, and you wanted everything from that part of the code on down to be tracked with a single promise.

Reply:
Basically it's only necessary if the method itself has the tracksTransactions decorator.